src/Model/BaseBlock.php

```diff
@@ -80,7 +80,7 @@ public function __toString()
         return sprintf('%s ~ #%s', $this->getName(), $this->getId());
     }
 
-    public function setName(string $name): void
+    public function setName(?string $name): void
```

Comment:
Why this change ?

Reply:
Because this setter restricts setting a `null` value (name = nullable) and conflicts with the documentation `@var string|null` unless this has been changed.

Reply:
We have multiple `@var string|null` in Sonata project.
It's generally `null` by default, but then we only can set a value, and never set `null` again.

My question is not clear. I meant "Is there a valid use case for setting `null` here ?". Did you find code setting `null` ?

Reply:
Thank you for clarifying. A use case would be if the `BaseBlock` model is used for a (custom) block where the `name` field is not required and can be left empty (i.e. is nullable). While this used to be working perfectly fine before this, it breaks now.

Is there any reason to not be able to change the value to its default in this case?

Reply:
> Is there any reason to not be able to change the value to its default in this case?

No, I'm not against this change ; I was just curious.